doc/getting_started/build_sw.md

```diff
@@ -111,6 +111,34 @@ You can see all tests available under a given directory using `bazel query`, e.g
 ```console
 bazel query 'tests(//sw/device/tests/...)'
 ```
+### Tags and wildcards
+
+TLDR: `bazelisk.sh test --test_tag_filters=-cw310,-verilator,-vivado,-jtag,-eternal,-broken --build_tag_filters=-vivado,-verilator //...`
+*Should* be able to run all the tests and build steps in OpenTitan that don't require optional setup steps, and
+
+You may find it useful to use wildcards to build/test all targets in the OpenTitan repository instead of individual targets.
+`//sw/...` is shorthand for all targets in `sw` that isn't tagged with manual.
+If a a target (a test or build artifact) relies on optional parts of the "Getting Started" guide they should be tagged so they can be filtered out and users can `bazelisk.sh test //...` once they filter out the appropriate tags.
+We maintain or use the following tags to support this:
+* `broken` is used to tag tests that are committed but should not be expected by CI or others to pass.
+* `cw310`, `cw310_test_rom`, `cw310_rom`, and `cw310_other` are used to tag tests that depend on a correctly setup cw310 "Bergen Board" to emulate OpenTitan.
```

Comment:
Could you clarify the difference between `cw310` and the other `cw310_*` tags? I'm wondering if it's defunct now. It looks it's defined as a required tag by `cw310_params()` in `opentitan_test.bzl`, but [`opentitan_functest()`](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;l=331-340;drc=4580a1765e7b5bd9a5f96c71f89e746e6c27c95d) will always add `cw310_rom` or `cw310_test_rom` for CW310 targets.

Reply:
Yeah, I guess since we can filter on the prefix, -cw310 works without having the tag so I'll just highlight that.

Reply:
The cw310 tag still exists and can still be used FWIW.